web/pdf_presentation_mode.js

```diff
@@ -97,14 +97,14 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
     /**
      * Switches page when the user scrolls (using a scroll wheel or a touchpad)
      * with large enough motion, to prevent accidental page switches.
-     * @param {number} delta - The delta value from the mouse event.
+     * @param {number} delta - The value from the mouse event in page units.
      */
     mouseScroll: function PDFPresentationMode_mouseScroll(delta) {
       if (!this.active) {
         return;
       }
       var MOUSE_SCROLL_COOLDOWN_TIME = 50;
-      var PAGE_SWITCH_THRESHOLD = 120;
+      var PAGE_SWITCH_THRESHOLD = 1.0;
```

Comment:
My setup might be atypical (these days), but I'm using a Logitech mouse which is set so that I get `evt.deltaY === +/- 3` in the `wheel` event for each scrollwheel "tick".

For me personally, I'd probably need `PAGE_SWITCH_THRESHOLD <= 0.20` to get a better page switching behaviour in presentation mode. Again, my setup might not be typical, so we probably need feedback from more users to come up with sensible defaults.